Pipfile

```diff
@@ -0,0 +1,21 @@
+[[source]]
+
+url = "https://pypi.python.org/simple"
+verify_ssl = true
+name = "pypi"
+
+
+[packages]
+
+requests = "*"
+configparser = "*"
+python-dateutil = "*"
+
+
+[dev-packages]
+
+
+
+[requires]
+
+python_version = "2.7"
```

Comment:
I'm confused by this line

Reply:
Feel free to remove this section, according to https://docs.pipenv.org/basics/#specifying-versions-of-python